packages/p4pp/p4pp.0.1.11/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis: "P4PP: Preprocessor for P4 Language"
+maintainer: "jnfoster@cs.cornell.edu"
+authors: ["Nate Foster <jnfoster@cs.cornell.edu>"]
+available: opam-version >= "2.1.0"
```

Comment:
Is there a particular reason for requiring opam >= 2.1?
```suggestion
```

Reply:
Oops, no. I think we were remove `opam depext` calls from some CI scripts and got overzealous.